.markdownlintignore

```diff
@@ -0,0 +1 @@
+/CHANGELOG.md
```

Comment:
Changelog had a few violations stemming from commit message, so these should not be fixed.